modules/claims_api/app/swagger/claims_api/forms/form_526_example.json

```diff
@@ -1,130 +1,126 @@
 {
-  "data": {
-    "type": "form/526",
-    "attributes": {
-      "veteran": {
-        "currentlyVAEmployee": false,
-        "currentMailingAddress": {
-          "addressLine1": "1234 Couch Street",
-          "addressLine2": "Apt. 22",
-          "city": "Portland",
-          "country": "USA",
-          "zipFirstFive": "12345",
-          "zipLastFour": "6789",
-          "type": "DOMESTIC",
-          "state": "OR"
-        },
-        "changeOfAddress": {
-          "beginningDate": "2018-06-04",
-          "addressChangeType": "PERMANENT",
-          "addressLine1": "1234 Couch Street",
-          "addressLine2": "Apt. 22",
-          "city": "Portland",
-          "country": "USA",
-          "zipFirstFive": "12345",
-          "zipLastFour": "6789",
-          "type": "DOMESTIC",
-          "state": "OR"
-        },
-        "homelessness": {
-          "pointOfContact": {
-            "pointOfContactName": "Jane Doe",
-            "primaryPhone": {
-              "areaCode": "123",
-              "phoneNumber": "1231234"
-            }
-          },
-          "currentlyHomeless": {
-            "homelessSituationType": "fleeing",
-            "otherLivingSituation": "other living situation"
-          }
-        }
+  "type": "form/526",
```

Comment:
Same here might need to keep this payload in tact and instead change the swagger section instead?